zipkin-anormdb/build.gradle

```diff
@@ -0,0 +1,20 @@
+// Database drivers
```

Comment:
How about this? https://github.com/openzipkin/zipkin/commit/e3619a765069c29b15bc2e3f75c906f4f12345fb